api/questions/question.js

```diff
@@ -54,7 +54,10 @@ function formatQuestionRes(question, answers) {
 
 function getQuestionsByCase(caseId, userId, options, jurisdiction) {
     return getHearingByCase(caseId, options)
-        .then(hearing => hearing.online_hearings[0] ? hearing.online_hearings[0].online_hearing_id : postHearing(caseId, userId, options, jurisdiction))
+        .then(hearing => {
+            console.log(JSON.stringify(hearing));
```

Comment:
should we be logging this?

Reply:
You're right. I left it in there for debugging. I'll update it

Reply:
just getting you back for all the time you do it to me :stuck_out_tongue_winking_eye:

Reply:
lol. Nah, it's cool. I've removed it.